File: Track/4. Array/9. Minimum adjacent difference in a circular array.java
class Solution
{
    public static int maxOccured(int L[], int R[], int n, int maxx)
    {
        int arr[]=new int[1000000];
        int sum=0,max=0,index=0;
        
        for(int i=0;i<n;i++)
        {
                arr[L[i]]++;
                arr[R[i]+1]--;
        }
        
        for(int i=0;i<1000000;i++)
        {
            sum+=arr[i];
            if(sum>max)
            {
                max=sum;
                index=i;
            }
        }
        return index;
    }
}

File: Track/4. Array/9. Minimum adjacent difference in a circular array.py
class Solution:
    def minAdjDiff(self,arr, n):
        min_dff = abs(arr[0]-arr[1])
        for i in range(1,len(arr)-1):
            if abs(arr[i]-arr[i+1]) < min_dff:
                min_dff = abs(arr[i]-arr[i+1])
            else:
                continue
        if min_dff > abs(arr[0]-arr[n-1]):
            min_dff = abs(arr[0]-arr[n-1])
        return min_dff
